packages/angular_devkit/build_angular/src/webpack/plugins/common-js-usage-warn-plugin.ts

```diff
@@ -121,5 +128,23 @@ export class CommonJsUsageWarnPlugin {
       // Non-scoped request ex: lodash/isEmpty -> lodash
       : rawRequest.split('/', 1)[0];
   }
+}
+
+// TODO_WEBPACK_5 ModuleGraph type stub
```

Comment:
I don't see any value added by having these stub types, considering they are used  below in a single method and you're casting.

Might as well cast inline.

Reply:
ok